frontend/src/store/modules/search.js

```diff
@@ -39,24 +39,29 @@ const categorizeResults = results => {
   return categorizedResults;
 };
 
+const componentTypeOrder = results =>
+  Object.entries(results)
+    .sort((a, b) => b[1].topScore - a[1].topScore)
+    .map(([k]) => k);
+
 const getters = {
   globalResultsEmpty: state =>
     Object.values(state.globalResults).reduce((s, r) => {
       const components = Object.keys(r).filter(k => k !== 'name');
       return s + components.reduce((t, c) => t + r[c].length, 0);
     }, 0) === 0,
 
-  categorizedGlobalResults: state => {
-    const results = categorizeResults(state.globalResults);
-    return Object.fromEntries(Object.entries(results).map(([k, v]) => [k, v.results]));
-  },
-
-  // eslint-disable-next-line no-unused-vars
-  categorizedGlobalResultsCount: (state, _getters) =>
+  categorizedGlobalResultsWithScores: state => categorizeResults(state.globalResults),
```

Comment:
Maybe just name it categorizedGlobalResults? Since it's with scores by default from now on. And maybe nice with some newlines between the  variables to improve readability?